code/src/NFluent/Messages/DateTimeOffsetRangeBlock.cs

```diff
@@ -0,0 +1,66 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+//  <copyright file="RangeBlock.cs" company="NFluent">
+//   Copyright 2019 Thomas PIERRAIN & Cyrille DUPUYDAUBY
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//       http://www.apache.org/licenses/LICENSE-2.0
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace NFluent.Messages
+{
+    using System;
+    using Helpers;
+
+    /// <summary>
+    ///     Class representing a range of value
+    /// </summary>
+    internal class DateTimeOffsetRangeBlock
+    {
+        private readonly DateTimeOffset referenceValue;
+        private readonly TimeSpan tolerance;
```

Comment:
This should be a Duration instead of a TimeSpan: the Duration class has been designed so that the preferred time unit is kept to ensure a easy to read output!